packages/web-runtime/src/components/ThemeSwitcher.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <oc-select
+      :model-value="currentThemeOrAuto"
+      :clearable="false"
+      :options="availableThemesAndAuto"
+      option-label="name"
+      @update:model-value="updateTheme"
+    />
+  </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent, ref, unref } from 'vue'
+import { useMessages, useThemeStore, WebThemeType } from '@ownclouders/web-pkg'
+import { storeToRefs } from 'pinia'
+import { useGettext } from 'vue3-gettext'
+
+export default defineComponent({
+  setup() {
+    const themeStore = useThemeStore()
+    const { showMessage } = useMessages()
+    const { $gettext } = useGettext()
+    const autoTheme = { name: $gettext('Auto (same as system)') }
```

Comment:
To enable reactivity on language change:

```suggestion
    const autoTheme = computed(() => ({ name: $gettext('Auto (same as system)') }))
```